### PR TITLE
Update fitbit.markdown to clarify secure setup

### PR DESCRIPTION
**Pull request by AddMoreLimes**
Added additional steps found in forum posts. This should help new users know that SSL is a prerequisite, because Fitbit won't allow unsecure communication to their APIs.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @AddMoreLimes, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21391).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by AddMoreLimes** [comment]
This may also be related to https://github.com/home-assistant/home-assistant.io/issues/20203

This issue may be that the user does not have SSL set up. There is no error message if you do not have SSL set up, which may cause confusion.

File: source/_integrations/fitbit.markdown [hunk 1]
```diff
@@ -12,6 +12,8 @@ ha_platforms:
 
 The Fitbit sensor allows you to expose data from [Fitbit](https://fitbit.com/) to Home Assistant.
 
+Prior to enabling the sensor, your Home Assistant instance must be enabled with SSL, using services such as [Home Assistant cloud](/cloud/) or [Duck DNS](/integrations/duckdns/) integrating Let's Encrypt. Fitbit will not allow insecure integrations. Your Fitbit account must also be registered as an Developer account at https://dev.fitbit.com and have a verified email address.
```
Comment (frenck) [inline source/_integrations/fitbit.markdown]: Please don't use bare URLs, instead link them in text.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fitbit.markdown (hunk 1 above)
Comment (AddMoreLimes) [reply source/_integrations/fitbit.markdown]: Developer portal link changed to text & link

**Review by frenck — APPROVED** [review]
Thanks, @AddMoreLimes 👍

**Pull request merged**